_posts/2018-06-26-why-you-need-https.md

```diff
@@ -0,0 +1,159 @@
+
+#### Google Chrome HTTP warning roll out
+
+Google (and closely mirroring behind, Firefox) has been getting more stringent in their display of the security implications of a site served over HTTP (in addition to sites misconfigured over HTTPS). They've shared details on the six-step roll out on [their general blog](https://blog.chromium.org/2018/05/evolving-chromes-security-indicators.html) as well as on a more technical, granular level on the [Chrome browser blog](https://www.chromium.org/Home/chromium-security/marking-http-as-non-secure).
+
+In January 2017, they began marking any site that collects a password field or credit card information, served over HTTP as subtly (grey text) not secure.
+
+![Chrome insecure message for HTTP]({{ site.url }}/assets/img/blog/jan-2017-http-not-secure.png)<br>
+ <span class="caption"> Google blog </span>
+
+Then, in October 2017, they tightened things up so that a site that collected _any_ form information over HTTP, would have the same "not secure" messaging. They added the more action-based aspect of showing the warning on the URL bar _when a user entered data into a form_. This is an especially obtrusive experience on mobile due to space constraints, which more deeply engages the user cognitively as to exactly what is unsafe about how they're interacting with the site.
+
+![Chrome insecure message for HTTP]({{ site.url }}/assets/img/blog/oct-2017-http-insecure.gif)<br>
+<span class="caption"> Google blog </span>
+
+Next, in July 2018, all HTTP sites will be marked as not secure.
+
+In September 2018, secure sites will be marked more neutrally, removing the green secure lock by default connoting a continuing expectation that HTTPS is the norm and no longer special.
+
+![Chrome insecure message for HTTP]({{ site.url }}/assets/img/blog/sep-2018-neutral-https.png)<br>
+<span class="caption"> Googleblog </span>
+
+In October 2018, any HTTP site that accepts any form fields will show affirmatively not secure with a <span style="color:#cc0000">**bold red label**</span>, much like a misconfigured HTTPS site does now.
+
+{% comment %} TODO: do bold red color in a less cheesy way? Ideas Anne? {% endcomment %}
+
+![Chrome insecure message for HTTP]({{ site.url }}/assets/img/blog/oct-2018-http-affirmative-insecure.gif)<br>
+<span class="caption"> Google blog </span>
+
+Though they haven't yet announced a date, Google intends to show affirmatively not secure for all HTTP sites. The drive is clearly to establish the norm that all the web traffic should be served over HTTPS and that outdated HTTP is not to be trusted. This is a pretty strong message that if Google has their way (which they usually do) HTTPS will inevitably be virtually mandatory. And inevitably in internet years, may be right around the corner.
+
+### HTTPS vastly improves security for you and your users
```

Comment:
I think this section is too far down in the article, and it goes into more technical detail than is necessary. I think it's a really important topic to cover, important enough that I think it should be the first item under "Why secure browsing matters." That said, I think it should be cut down a lot to just cover a concise explanation of why HTTP is insecure. The "what encryption does for you" section is particularly accessible - could you consider trimming down the other sections and removing as much technical detail as possible, especially in the "data integrity" section?